docs/reference/upgrade.md

```diff
@@ -21,8 +21,43 @@ If you're upgrading, check whether the [release](https://github.com/treeverse/la
 Starting with version <TBD.X.X>, lakeFS has transitioned from using a PostgreSQL based database implementation to a Key-Value datastore interface supporting
 multiple database implementations. More information can be found [here](link-to-lakefs-on-kv-documentation).  
 Users upgrading from a previous version of lakeFS must pass through the KV migration version (<TBD.X.X>) before upgrading to newer versions of lakeFS.
-The migration process is identical to the process described [here](#lakefs-0300-or-greater) and ensures a clean transition from the old metadata database model to the new one.
 
+#### Migration Steps
+For each lakeFS instance currently running with the database
+1. Modify the `database` section under lakeFS configuration yaml:
+   1. Add `type` field with `"postgres"` as value
+   2. Copy the current configuration parameters to a new section called `postgres`
+   ```yaml
+   ---
+   database:
+    type: "postgres"
+    connection_string: "postgres://localhost:5432/postgres?sslmode=disable"
+    max_open_connections: 20
+   
+    postgres:
+      connection_string: "postgres://localhost:5432/postgres?sslmode=disable"
+      max_open_connections: 20
```

Comment:
Why should I use the same configuration twice? It adds friction which we can avoid in the server side pretty easily, no?

Reply:
This will require us to 'redirect' all of our configuration to the new structure - we will need to modify production code, unit tests and system test to take the database URI from the new configuration path. Should I open a PR for it?

Reply:
This can potentially break all of our production environments